examples/analysis/geospatial/proximity_metric.py

```diff
@@ -0,0 +1,50 @@
+"""
+Proximity to raster or vector
+=============================
+
+This example demonstrates the calculation of proximity distances to a raster or vector using :func:`~geoutils.Raster.proximity`.
+"""
+# sphinx_gallery_thumbnail_number = 2
```

Comment:
This line gets rendered, it probably shouldn't be?

Reply:
Solved in [9d487d0](https://github.com/GlacioHack/geoutils/pull/351/commits/9d487d0a85fe2acffbeb4b75d2e3765d23090f66)